Userland/Libraries/LibJS/Runtime/Temporal/ISO8601.cpp

```diff
@@ -786,6 +1004,120 @@ bool ISO8601Parser::parse_time_spec()
     return true;
 }
 
+// https://tc39.es/proposal-temporal/#prod-TimeHourMinuteBasicFormatNotAmbiguous
+bool ISO8601Parser::parse_time_hour_minute_basic_format_not_ambiguous()
+{
+    // TimeSpecWithOptionalTimeZoneNotAmbiguous :
```

Comment:
Shouldn't this be `TimeHourMinuteBasicFormatNotAmbiguous`?

Reply:
Yep, fixed!